argoverse/utils/cv2_video_utils.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+import cv2
+import numpy as np
+
+"""
+Python-based utilities to avoid blowing up the disk with images, as FFMPEG requires.
```

Comment:
Does this replace this dependency? https://github.com/argoai/argoverse-api/blob/1dfc03ee8dc9b338eac6c86e21d7b999b59e2498/argoverse/visualization/generate_sequence_videos.py#L108

Reply:
I didn't realize we had this dependency tbh (it's not in our dependency list for some reason), I'll take a look

Reply:
What were your thoughts on this?

Reply:
haven't played around with moviepy before, but we should check it out more: https://zulko.github.io/moviepy/